api/src/pcapi/core/offerers/api.py

```diff
@@ -1433,28 +1434,108 @@ def list_offerers_to_be_validated(
     from_datetime: datetime | None = None,
     to_datetime: datetime | None = None,
 ) -> sa.orm.Query:
-    query = offerers_models.Offerer.query.options(
-        sa.orm.joinedload(offerers_models.Offerer.UserOfferers).joinedload(offerers_models.UserOfferer.user),
-        sa.orm.joinedload(offerers_models.Offerer.tags)
-        .joinedload(offerers_models.OffererTag.categories)
-        .load_only(offerers_models.OffererTagCategory.name),
-        sa.orm.joinedload(offerers_models.Offerer.action_history).joinedload(history_models.ActionHistory.authorUser),
-        sa.orm.joinedload(offerers_models.Offerer.managedVenues)
-        .load_only(offerers_models.Venue.name, offerers_models.Venue.siret)
-        .joinedload(offerers_models.Venue.collectiveDmsApplications)
-        .load_only(
-            educational_models.CollectiveDmsApplication.state,
-            educational_models.CollectiveDmsApplication.lastChangeDate,
-        ),
+    # Aggregate tags as a json dictionary returned in a single row (joinedload would fetch 1 result row per tag)
+    # For a single offerer, column value is like:
+    # [{'name': 'top-acteur', 'label': 'Top Acteur'}, {'name': 'culture-scientifique', 'label': 'Culture scientifique'}]
+    tags_subquery = (
+        sa.select(
+            sa.func.jsonb_agg(
```

Comment:
Ah je connaissais pas ça, tu les mets les valeurs dans un json pour pouvoir faire `tags_subquery.name` dans les templates jinja?

Reply:
Oui c'est ça, par contre le nom `tags_subquery` n'est qu'un nom de variable, ensuite la colonne s'appelle `tags`, du fait de `tags_subquery.label("tags")`.
Dans le jinja, je liste ainsi les tags :
`{{ item.tags | map(attribute="label") | join(", ") | escape }}`
Cette syntaxe correspond à `", ".join(tag["label"] for tag in tags)` en Python